docs/running_a_server/whitelisting.md

```diff
@@ -0,0 +1,163 @@
+---
+id: whitelisting
+title: Setting Up Whitelist
+slug: /whitelisting
+hide_title: true
+hide_table_of_contents: true
+sidebar_label: Setting up Whitelist on Java/Bedrock
+description: How to setup Whitelist on Java/Bedrock
+keywords:
+  - Minecraft
+  - Whitelist
+  - Security
+  - Secure
+  - Server 
+image: https://bloom.host/assets/images/logo.png
+---
+<div class="text--center">
+<img src="https://bloom.host/logo-white.svg" alt="logo" height="50%" width="50%"/>
+<h1>Setting Up Whitelist</h1>
+</div>
+
+
+:::caution
```

Comment:
Could we move this to below the `What Is Whitelisting` section? Just so the page doesn't open with a warning 😄

Reply:
Done! 😄